package.json

```diff
@@ -31,5 +31,17 @@
       "last 1 firefox version",
```

Comment:
With all my respect, we need to add `prop-types` 🙏 
![image](https://user-images.githubusercontent.com/31897434/90679180-b41c3100-e257-11ea-9e25-3eaacd4bdc95.png)
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896085/

Reply:
Thanks for the review, for some reason I missed that.